pkgs/applications/science/electronics/bitscope/common.nix

```diff
@@ -0,0 +1,67 @@
+{ atk
+, buildFHSUserEnv
+, cairo
+, dpkg
+, fetchurl
+, gdk_pixbuf
+, glib
+, gtk2-x11
+, makeWrapper
+, pango
+, stdenv
+, writeScriptBin
+, xorg
+}:
+
+{ src, toolName, version, ... } @ attrs:
+let
+  wrapBinary = libPaths: binaryName: ''
+    wrapProgram "$out/bin/${binaryName}" \
+      --prefix LD_LIBRARY_PATH : "${stdenv.lib.makeLibraryPath libPaths}"
+  '';
+  pkg = stdenv.mkDerivation (rec {
+    inherit (attrs) version src;
+
+    name = "${toolName}-${version}";
+
+    meta = with stdenv.lib; {
+      homepage = http://bitscope.com/software/;
+      license = licenses.unfree;
+      platforms = [ "i686-linux" "x86_64-linux" ];
```

Comment:
We no longer support `i686`. Are you sure you want to provide `i686` packages?

Reply:
I just took it out. Not sure at all and if it isn't supported anymore I don't see the point keeping it there :wink:. Thanks for pointing that out.

Reply:
so all the
```
else if stdenv.system == "i686-linux" then fetchurl
```
can be dropped as well then, right?

Reply:
A few minutes after the push it dawned upon me that I had those obsolete `stdenv.system` checks in there. They're gone now. :wink: